eagledns/src/main/java/se/unlogic/standardutils/string/StringUtils.java

```diff
@@ -91,15 +92,9 @@ public static String removeHTMLTags(String sequence) {
 		return sequence.replaceAll("\\<.*?>", "");
 	}
 
-	public static String parseUTF8(String encodedString) {
+	public static String parseUTF8(String encodedString) throws UnsupportedEncodingException {

```

Comment:
@CathrineTeng  Please look at the above comment about UnsupportedEncodingException change.